BugsnagPerformance/Assets/BugsnagPerformance/Scripts/Internal/TracePayload.cs

```diff
@@ -20,7 +20,7 @@ public class TracePayload
 
         private string _jsonbody;
 
-        public TracePayload(ResourceModel resourceModel, List<Span> spans)
+        public TracePayload(ResourceModel resourceModel, List<Span> spans, bool probabilityOverride)
```

Comment:
Can we refactor any references to an override?

```suggestion
        public TracePayload(ResourceModel resourceModel, List<Span> spans, bool isFixedSamplingProbability)
```